benchmarks/generated_benchmark_suites.cmake

```diff
@@ -15,34 +15,367 @@ set(_IREE_ARTIFACTS_DIR "${_ROOT_ARTIFACTS_DIR}/iree")
 ################################################################################
 # Below is generated by build_tools/benchmarks/suites/cmake_rule_generator.py  #
 ################################################################################
+# Fetch the model from "https://storage.googleapis.com/iree-model-artifacts/deeplabv3.tflite"
+iree_fetch_artifact(
```

Comment:
Not this PR, but I'd love to see us move to making fetching things from the network *not* part of the build system and instead an explicit separate step that usually only needs to be done once and then the files can sit on disk outside the build directory and only be refreshed when necessary (we can record the digest). I suppose we *could* still include that in the build system, like by having a parameter for the archive directory and putting it outside the build directory. And then doing hash comparisons (or naming via hash).

Reply:
Tracked with #10525